app/views/admin/users/widgets/_credits.html.erb

```diff
@@ -0,0 +1,11 @@
+<header class="mb-4">
+  <h2 class="crayons-subtitle-2 mb-1 flex items-center">
+    <%= crayons_icon_tag(:wallet, title: "Wallet", class: "mr-2") %>
+    Credits
+  </h2>
+  <p class="color-secondary">Credits enable users to purchase classified listings from your site. <a href="#" class="c-link c-link--branded">Learn more</a></p>
+</header>
+<div class="flex justify-between items-center">
+  <p class="fs-2xl fw-heavy"><%= @user.unspent_credits_count %></p>
+  <button class="c-btn c-btn--secondary whitespace-nowrap" type="button">Adjust balance</button>
```

Comment:
This needs to trigger a modal as well

cc @nickytonline

Reply:
Done.